base/install_modules.sh

```diff
@@ -65,3 +65,7 @@ EPICS_BASE
 install_from_github -i epics-modules ether_ip ETHER_IP $ETHER_IP_VERSION "
 EPICS_BASE
 "
+
+install_from_github  epics-modules iocStats IOCADMIN $IOCSTATS_VERSION "
+EPICS_BASE
+"
```

Comment:
The expected name for this module is `DEVIOCSTATS`. That's the library name, it won't work otherwise for other users.

`IOCADMIN` is only referred to in https://github.com/slac-epics-modules/ipmiComm/blob/3611042d5ad8394bacd073481c6b510fa2805199/configure/RELEASE_SPEAR#L22 , but that's not used for the build, as far as I can see, and it's SPEAR specific (they are even using the custom name from SLAC's repo: https://github.com/slac-epics/iocAdmin ), as far as I could see it's not actually used. Did you get any errors without it?

The commit message is wrong. There is no iocAdmin library, [only `devIocStats`](https://github.com/search?q=repo%3Aepics-modules%2FiocStats%20LIBRARY_IOC&type=code). I think the commit should ideally be simply addition of the module, and it shouldn't require any explanations as to why...